### ISSUE TITLE
Archive Index issue

### ISSUE THREAD

**Issue by RayGX** [justingit/dada-mail#950]
The Archive Index link (see image below) which appears at the bottom when a list member is viewing archived messages does not appear to present an index.  When clicked, I am seeing the exact same presentation of messages that I see when I first get to the Archived Messages page, not a listing of messages to select from (which would be my interpretation of an index).

RC

![ArcIndex](https://user-images.githubusercontent.com/10082909/88820522-056c5f80-d187-11ea-9060-2ce676b89404.jpg)

**Comment by justingit** [comment]
The "Archive Index" screen would be the same as what you're calling the, "Archived Message" screen - so for example: 

https://dadamailproject.com/cgi-bin/dada/mail.cgi/archive/dada_announce

Perhaps the confusion is that the list screen also has an index of the archive messages: 

https://dadamailproject.com/cgi-bin/dada/mail.cgi/list/dada_announce

And I guess "Archive Index" seem a little awkwardly worded. 



For a (new) Archive Index, would you be expecting just a huge list of archive messages available to view?

**Comment by RayGX** [comment]
Yes, essentially a list where each single line would contain the Subject and Sent date items for each message (From would seem to be redundant as it would always be From the list being viewed).  Clicking an item in this listing would then show the respective message.  There is such a listing if I log in as Administrator and select Archives > View from the menu, although it also shows a lot of code and is not a "clean" view.

RC
